hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/ContractsV_0_32Module.java

```diff
@@ -64,7 +65,12 @@ public interface ContractsV_0_32Module {
     @ContractsModule.V_0_32
     static BiPredicate<Address, MessageFrame> provideAddressValidator(
```

Comment:
This is no longer address validator, if we check if it's valid and attempt to lazy create.  It's an existence checker.  We should re-name it to something like `provideAddressExistsFunction`.

Reply:
Will resolve in a future PR